Makefile

```diff
@@ -0,0 +1,48 @@
+artifact_name       := company-documents.web.ch.gov.uk
```

Comment:
There is no test target in this file.

Reply:
Mentioned in slack PM, removed from earlier comment. Closing :)